src/sentry/integrations/utils/commit_context.py

```diff
@@ -111,8 +124,23 @@ def find_commit_context_for_event_all_frames(
     return (selected_blame, selected_install)
 
 
-def is_date_less_than_two_months(date: datetime) -> bool:
-    return date > datetime.now(tz=timezone.utc) - timedelta(days=60)
+def is_too_young(group_first_seen: datetime, commit_date: datetime) -> bool:
+    """
+    Returns True if the commit happened after the issue was first seen.
+    These commits cannot have caused the issue.
+    """
+    return commit_date > group_first_seen
+
+
+def is_too_old(
+    group_first_seen: datetime, commit_date: datetime, max_age_days: int = MAX_COMMIT_AGE_DAYS
+) -> bool:
+    """
+    Returns True if the commit is too old to be considered relevant.
+    Commits older than max_age_days before the issue was first seen are ignored.
+    """
+    time_threshold = group_first_seen - timedelta(days=max_age_days)
+    return commit_date < time_threshold
```

Comment:
nit: IMO, i would just inline these two functions. they are idiomatic enough to where i'm not sure if we're getting enough bang-for-our-buck extracting the logic into separate functions.

(see [refactoring](https://dl.ebooksworld.ir/motoman/Refactoring.Improving.the.Design.of.Existing.Code.2nd.edition.www.EBooksWorld.ir.pdf) 115 [Inline Function](https://gist.github.com/cs-cordero/3799f26699bdecdb286fd719f08122af#inline-function-115))